### PR TITLE
CLDR-13263 Constructed-value-related clean-up of CLDRFile

### PR DESCRIPTION
**Pull request by btangmu**
-Move ten CLDRFile.getName functions and related subroutines/data to bottom of CLDRFile.java in preparation for refactoring as separate class

CLDR-13263

- [ ] This PR completes the ticket.

### CODE REVIEW

**Comment by btangmu** [comment]
There are no changes in this PR other than the ordering of code, and very minor formatting/comments.

I have to ask that you take my word for this, along with trusting the tests, since unfortunately git diff doesn't show the changes in a straightforward manner.

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java [hunk 1]
```diff
@@ -2251,679 +2251,260 @@ public static String getCode(String path) {
...
-     * @param type
-     * @param code
-     * @param codeToAlt - if not null, is called on the code. If the result is not null, then that is used for an alt value.
-     * If the alt path has a value it is used, otherwise the normal one is used. For example, the transform could return "short" for
-     * PS or HK or MO, but not US or GB.
-     * @return
+     * @param type a string such as "language", "script", "territory", "region", ...
+     * @return the corresponding integer
      */
-    public String getName(int type, String code, Transform<String, String> codeToAlt) {
```
Comment (btangmu) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java:2268]: all getName are moved to the bottom of this file

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java [hunk 2]
```diff
@@ -2251,679 +2251,260 @@ public static String getCode(String path) {
...
-        if (codeToAlt != null) {
-            String alt = codeToAlt.transform(code);
-            if (alt != null) {
-                String altPath = path + "[@alt=\"" + alt + "\"]";
-                result = getStringValueWithBaileyNotConstructed(altPath);
-            }
-        }
-        if (result == null) {
-            result = getStringValueWithBaileyNotConstructed(path);
+    public static int typeNameToCode(String type) {
```
Comment (btangmu) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java:2258]: there is no change to typeNameToCode; unfortunately git diff is confused

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java [hunk 3]
```diff
@@ -2251,679 +2251,260 @@ public static String getCode(String path) {
...
-                } else if (type == SCRIPT_NAME) {
-                    result = getLstrFallback("script", code);
-                }
+        for (int i = 0; i < LIMIT_TYPES; ++i) {
+            if (type.equalsIgnoreCase(getNameName(i))) {
+                return i;
             }
         }
-        return result;
+        return -1;
```
[the remark this answers is not in the document]
Comment (btangmu) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/CLDRFile.java:2267]: and likewise below

**Review by srl295 — APPROVED** [review]
- Review Hint: used github.dev ( hit `.` on your keyboard), the diffs were a lot more readable.
- Review comment:  _How did all of this end up in CLDRFile in the first place? It's quite the kitchen sink!_

**Pull request merged**